pytorch_lightning/loggers/neptune.py

```diff
@@ -523,6 +523,17 @@ def after_save_checkpoint(self, checkpoint_callback: "ReferenceType[ModelCheckpo
             file_names.add(model_name)
             self.experiment[f"{checkpoints_namespace}/{model_name}"].upload(key)
 
+        # log best model path
+        if checkpoint_callback.best_model_path:
+            self.experiment[
+                self._construct_path_with_prefix("model/best_model_path")
+            ] = checkpoint_callback.best_model_path
+
+            model_name = self._get_full_model_name(checkpoint_callback.best_model_path, checkpoint_callback)
+            file_names.add(model_name)
+            self.experiment[f"{checkpoints_namespace}/{model_name}"].upload(
+                checkpoint_callback.best_model_path)
```

Comment:
Can we enable this optionally with a flag? I think this flag should also default to `False` to replicate current behaviour.

Background: There are people having only a limited bandwith during training

Reply:
Hey @justusschock,
We already have `log_model_checkpoints` which is set to `True` at default. In the case of the user calling ModelCheckpoint directly, it should work pretty fine with uploading model checkpoints (before this bugfix). We're trying to document in all known places that `log_model_checkpoints=True` is equal to uploading every checkpoint that was saved by a Trainer. In such a context, I think that we're filling the gap instead of breaking some expected behavior. So I don't think it's necessary to add any new flag :wink: